### DISCUSSION TITLE
Welcome to dlite Discussions!

### DISCUSSION THREAD

**Discussion by renaudpawlak** [cincheo/daquota#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HFolkertH** [comment]
hi Renaud,

Greetings from le pays-bas.
I am now just experimenting with dlite for private use. But who knows...

kind regards
Folkert

**Reply by renaudpawlak** [reply]
Great! Private use is clearly a use case (too)! Best luck with your developments and I hope you will have fun with DLite.
